types/src/multihash.rs

```diff
@@ -0,0 +1,214 @@
+use multihash_derive::{Hasher, MultihashDigest};
+use nmt_rs::simple_merkle::tree::MerkleHash;
+use nmt_rs::NamespaceMerkleHasher;
+
+use crate::byzantine::MULTIHASH_SHA256_NAMESPACE_FLAGGED_CODE;
+use crate::nmt::{
+    Namespace, NamespacedHashExt, NamespacedSha2Hasher, NAMESPACED_HASH_SIZE, NS_SIZE,
+};
+use crate::{Error, Result};
+
+pub type NamespacedHash = nmt_rs::NamespacedHash<NS_SIZE>;
```

Comment:
can we move this alias to nmt mod and import it there?

Reply:
Good idea, moved